spec/sections/usage.adoc

```diff
@@ -101,6 +101,18 @@ time down to minute precision.
 <5> Required string containing an ISO 8601 date indicating the end temporal extent of file. String must include full
 time down to minute precision. If the data is a single point-in-time the end date should match the start date.
 
+[[resource-defaults, Resource Defaults]]
+==== Resource Defaults
+At a minimum, the _executor_ MUST provide at least the resources indicated by a Seed manifest at run time. If the
+resource requirement specified by the manifest is below the minimum amount allowed by the _executor_ it MAY float the
+requirement to that value. _Developers_ SHOULD specify the `cpu`, `mem` and `disk` requirements of their Seed job, but
+if these are not set, the _executor_ is free to allocate minimal defaults.
+
+For the `sharedMem` reserved resource, the _executor_ MAY make use of the Docker run argument `shm-size` to provide the
```

Comment:
Change 'float' to 'increase' and 'MAY' to 'SHOULD' when taking about shared memory Docker param.